### PR TITLE
Accessibility

### PR DESCRIPTION
**Pull request by Obote**
### In this branch, the following adaptations for accessibility were made:

- **Changed the title** - to briefly describe the title of the page;
- Image text alternatives - no issue found;
- Text headings - no issue found;
- Color contrast - no issue found;
- Resize - no issue found;
- **Interaction** - fix interaction for better keyboard accessibility;
- Moving content - no issue found;
- Multimedia - no issue found;
- The basic structure of the page - no issue found.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,7 +5,16 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
-    <title>Obote</title>
```
[the remark this answers is not in the document]
Comment (carlos-ssh) [reply index.html:8]: **Kindly work on these changes**
- Would be great to have a title like these examples:
Obote's portifolio
Obote full-stack webdeveloper

File: index.html [hunk 2]
```diff
@@ -1,173 +1,272 @@
...
+          <li><a href="#about">About</a></li>
+          <li><a href="#contact">Contact</a></li>
+        </ul>
+        <img
+          class="msg-icon"
+          src="./images/Icons/Union.svg"
+          alt="message icon"
+        />
+        <button class="hamburger">
+          <img src="./images/Icons/menu-icon.png" alt="" />
```
Comment (carlos-ssh) [inline index.html:39]: Provide text to your ``alt`` parameter.

File: index.html [hunk 3]
```diff
@@ -1,173 +1,272 @@
...
-                <li class="border-right padding-right">css</li>
-                <li class="border-right padding-right">html</li>          
-                <li class="border-right padding-right">bootstrap</li>         
-                <li class="padding-right">Ruby</li>
+              <li class="border-right padding-right">css</li>
+              <li class="border-right padding-right">html</li>
+              <li class="border-right padding-right">bootstrap</li>
+              <li class="padding-right">Ruby</li>
             </ul>
             <button type="button" class="project-button">See Project</button>
```
Comment (carlos-ssh) [inline index.html:141]: Kindly work on the requested design in your buttons, hover effects. Follow the design guidelines for it on each button.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @Obote , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ZeenatLawal — APPROVED** [review]
Hi @Obote and @nicupop729, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 
6vEo


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**